src/configuration/locales/fr/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Réponder au questionnaire!"
+  "...Loading": "...Chargement",
+  "Beekeeper": "Apiculteur",
+  "Beginner": "Débutant",
+  "Billed: {{value}}": "Payant: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Expérience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Espace disponible",
+  "Garden": "Jardin",
+  "HAPPYCULTEUR © COPYRIGHT 2019. ALL RIGHTS RESERVED.": "Happyculteur © COPYRIGHT 2019. TOUS DROITS RÉSERVÉS.",
+  "Happyculteur partner!": "Partenaire Happyculteur!",
+  "Individual": "Particulier",
+  "Install": "Installer des ruches",
+  "Interest:": "Intérêt:",
+  "LEGAL NOTICE": "MENTIONS LEGALES",
+  "Learn": "Apprendre",
+  "Meadow": "Prairie",
+  "Medium": "Moyen",
+  "Public:": "Publique:",
+  "Roof": "Toit",
+  "SPECIAL THANKS": "Remerciement spéciaux",
+  "Scroll to load more": "Défiler vers le bas pour charger plus",
+  "Sell": "Vendre",
+  "Since: {{seniority}} years": "Depuis: {{seniority}} years",
+  "Size: {{size}}": "Taille: {{size}}",
+  "Space": "Espace",
+  "Take the survey!": "Réponder au questionnaire!",
```

Comment:
`Répondre au questionnaire!`
But actualyly in the app it should be `Enregistrez-vous`, fast forwarding to the previous meeting with Anastasia and Elisa.

Reply:
Oups I corrected the typo in the PR: `Réponder`=> `Répondez`

Reply:
I will let the survey question for now until we have a proper backen to handle user registration ;)